sos/report/plugins/openshift_ovn.py

```diff
@@ -20,18 +20,30 @@ class OpenshiftOVN(Plugin, RedHatPlugin):
     profiles = ('openshift',)
 
     def setup(self):
+        all_logs = self.get_option("all_logs")
+
         self.add_copy_spec([
             "/var/lib/ovn/etc/ovnnb_db.db",
             "/var/lib/ovn/etc/ovnsb_db.db",
-            "/var/lib/openvswitch/etc/keys",
-            "/var/log/openvswitch/libreswan.log",
-            "/var/log/openvswitch/ovs-monitor-ipsec.log"
-        ])
-        # Collect ovn interconnect specific files if exists.
+            "/var/lib/openvswitch/etc/keys"
+        ], sizelimit=300)
+
+        # Collect ovn interconnect specific db files if exists.
         self.add_copy_spec([
             "/var/lib/ovn-ic/etc/ovnnb_db.db",
             "/var/lib/ovn-ic/etc/ovnsb_db.db"
-        ])
+        ], sizelimit=300)
+
+        if not all_logs:
+            self.add_copy_spec([
+                "/var/log/openvswitch/libreswan.log",
+                "/var/log/openvswitch/ovs-monitor-ipsec.log"
+            ])
+        else:
+            self.add_copy_spec("/var/log/openvswitch")
+
+        # Collect libovsdb logs for ovn interconnect setup.
+        self.add_copy_spec("/var/lib/ovn-ic/etc/libovsdb*log*", sizelimit=600)
```

Comment:
imo, I'm still keen to use all_log for all collections and `.log` if that has not been asked for. But, happy for other people to chime in here